dashboard/test/lib/pd/payment/payment_calculator_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+module Pd::Payment
+  class PaymentCalculatorTest < ActiveSupport::TestCase
+    self.use_transactional_test_case = true
+
+    test 'Raise error if workshop is not ended' do
+      workshop = create(:pd_workshop, course: Pd::Workshop::COURSE_CSF, enrolled_and_attending_users: 20)
```

Comment:
not sure if this slipped through the cracks, or you decided not to change it, but i'd remove the `enrolled_and_attending_users` for this test since the users are irrelevant. Your call